src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -245,7 +252,8 @@ HttpStatusCode mongoQueryContext
   QueryContextResponse*                responseP,
   const std::string&                   tenant,
   const std::vector<std::string>&      servicePathV,
-  std::map<std::string, std::string>&  uriParams
+  std::map<std::string, std::string>&  uriParams,
+  long long*                           countP
```

Comment:
Yes.
For v2 I need that number to be able to add the HTTP header X-Total-Count.

NTC